cpp/src/parquet/printer.cc

```diff
@@ -142,6 +142,15 @@ void ParquetFilePrinter::DebugPrint(std::ostream& stream, std::list<int> selecte
     stream << "--- Total Bytes: " << group_metadata->total_byte_size() << " ---\n";
     stream << "--- Total Compressed Bytes: " << group_metadata->total_compressed_size()
            << " ---\n";
+    auto sorting_columns = group_metadata->sorting_columns();
```

Comment:
Nit
```suggestion
    const auto& sorting_columns = group_metadata->sorting_columns();
```

Reply:
```c++
std::vector<SortingColumn> RowGroupMetaData::sorting_columns() const
```

Seems that this returns a vector directly, and just auto is ok here...